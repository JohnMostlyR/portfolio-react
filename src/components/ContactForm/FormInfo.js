import React from 'react';
import {FormattedMessage} from 'react-intl';
import styled from 'styled-components';
import FontAwesome from 'react-fontawesome';

const Wrapper = styled.div`
  padding-bottom: 1rem;
`;

const FormInfoItemsList = styled.ul`
  list-style: none;
`;

const FormInfo = () => (
    <Wrapper>
      <FormInfoItemsList>
        <li>
          <FontAwesome name="info-circle"/> <span><FormattedMessage
            id={'portfolio.page.contact.requirement.one'}
            defaultMessage={'Please fill in all fields'}/></span>
        </li>
      </FormInfoItemsList>
    </Wrapper>
);

export default FormInfo;