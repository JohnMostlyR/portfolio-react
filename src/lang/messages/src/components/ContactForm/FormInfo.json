[
  {
    "id": "portfolio.page.contact.requirement.one",
    "defaultMessage": "Please fill in all fields"
  }
]