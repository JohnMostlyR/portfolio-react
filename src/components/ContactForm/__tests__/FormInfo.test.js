import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

import FormInfo from '../FormInfo';

describe('FormInfo', () => {
  it('should render and match the snapshot', () => {
    const wrapper = shallow(<FormInfo/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});