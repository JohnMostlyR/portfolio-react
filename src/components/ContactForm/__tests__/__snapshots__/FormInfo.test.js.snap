// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormInfo should render and match the snapshot 1`] = `
<styled.div>
  <styled.ul>
    <li>
      <FontAwesome
        name="info-circle"
      />
       
      <span>
        <FormattedMessage
          defaultMessage="Please fill in all fields"
          id="portfolio.page.contact.requirement.one"
        />
      </span>
    </li>
  </styled.ul>
</styled.div>
`;
